.storybook/test-data.js

```diff
@@ -13,6 +13,103 @@ const state = {
     protocol: 'https:',
     url: 'https://metamask.github.io/test-dapp/',
   },
+  networkList: [
+    {
+      blockExplorerUrl: "https://etherscan.io",
+      chainId: "0x1",
+      iconColor: "#29B6AF",
```

Comment:
Try to refrain from using static hex values if possible. That way we can maintain consistent colors across the app even in test data. It will also help if we decide to update these network colors some day we only need to update them in one place
```suggestion
      iconColor: 'var(--mainnet)',
```

Reply:
@georgewrmarshall Oh yeah, my bad! Sorry for using static hex values for `iconColor` in `test-data.js`! 
Changes done [here](https://github.com/MetaMask/metamask-extension/pull/13560/commits/8fb47ff2cebbf883b16f1137e43b43956ad98571)